### PR TITLE
Update partner portal link across the site

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated link as per docs: [1](https://docs.google.com/document/d/12tpcvTBOjwCGzqrGMFC0ucYCGATJJn8Tbl2SHUcgf6k/edit), [2](https://docs.google.com/document/d/1q6dkqQ0zcshrfOmdJA_bAip73EaMSEyXAn8bAD7GqNw/edit)
- Ran djlint

## QA

- View the site locally in your web browser at: 
  - https://canonical-com-1243.demos.haus/partners
  - https://canonical-com-1243.demos.haus/partners/channel-and-reseller
- See that the link has been updated in the above pages as well as on the partners nav

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10539

### CODE REVIEW

File: templates/partial/_standard-navigation.html [hunk 1]
```diff
@@ -61,7 +89,8 @@
               <a href="/partners/executive-summit" class="p-navigation__dropdown-item">Partner Executive Summit</a>
             </li>
             <li>
-              <a href="https://partner-portal.canonical.com/" class="p-navigation__dropdown-item">Log into the partner portal</a>
+              <a href="https://partner.canonical.com"
```
Comment (mtruj013) [inline templates/partial/_standard-navigation.html:92]: This is the only non-linting change to this file

File: templates/partners/channel-and-reseller.html [hunk 2]
```diff
@@ -4,115 +4,125 @@
...
-          <p class="p-heading--5">
-            Access to Canonical partner portal
+          <p>
+            Canonical is an industry leader in OpenStack private clouds, the reference platform for all major public clouds, and the open source software operating system that runs from the desktop and all internet connected things. Our success helps our partners build a business around our technology.
           </p>
         </div>
-        <div class="col-6 col-medium-3">
-          <p>We host a partner portal with product and solutions collaterals, agreed pricing, incentives and a deal registration system.</p>
-        </div>
+        <a href="https://partner.canonical.com" class="p-button--positive"><span>Login to the partner portal</span></a>
```
Comment (mtruj013) [inline templates/partners/channel-and-reseller.html:39]: This is the only non-linting change to this file

File: templates/partners/index.html [hunk 3]
```diff
@@ -2,146 +2,160 @@
...
-        </div>
-        <div class="col-6">
-          <p>Channel partners ensure that local experts in Ubuntu and all Canonical products can be found on every continent</p>
-        </div>
-      </div>
-      <div class="row">
-        <hr class="p-rule">
+      <div class="col-medium-4 col-9">
+        <p>
+          Existing partner? <a href="https://partner.canonical.com">Log in to the partner portal</a>
```
Comment (mtruj013) [inline templates/partners/index.html:41]: This is the only non-linting change to this file

File: templates/partners/partial/_co-sell-with-canonical.html [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
     <div class="col">
-      <p>Open source technology is growing in demand. We want you to benefit from this unique opportunity, and take advantage of the investment Canonical is making in its partner community. We are committed to giving you the tools and programs to succeed, from discounts and financial incentives, to sales and technical support teams.</p>
-      <hr class="p-rule">
-      <p><a href="https://partner-portal.canonical.com/">Login to the partner portal</a></p>
+      <p>
+        Open source technology is growing in demand. We want you to benefit from this unique opportunity, and take advantage of the investment Canonical is making in its partner community. We are committed to giving you the tools and programs to succeed, from discounts and financial incentives, to sales and technical support teams.
+      </p>
+      <hr class="p-rule" />
+      <p>
+        <a href="https://partner.canonical.com">Login to the partner portal</a>
```
Comment (mtruj013) [inline templates/partners/partial/_co-sell-with-canonical.html:19]: This is the only non-linting change to this file

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**